py-polars/polars/_utils/udfs.py

```diff
@@ -978,4 +978,21 @@ def warn_on_inefficient_map(
             )
 
 
-__all__ = ["BytecodeParser", "warn_on_inefficient_map"]
+def get_dynamic_lib_location(plugin_location: str | Path) -> str:
```

Comment:
Let's just include this as a private function in the plugins module. It's not used in multiple places, so it can live next to the code that uses it.